docs/development.md

```diff
@@ -0,0 +1,27 @@
+# Solidus Starter Frontend development information
+This document aims to give some extra information for developers that are going 
+to contribute to our `solidus_starter_frontend` component.
+
+## Solidus Compare tool
+`solidus_compare` is a tool that we created to keep track of the changes made to
+`solidus_frontend`, which we used as source project in the beginning.
```

Comment:
Maybe we can link to it?

Reply:
Good point.
Push forced a fixup to add the link to https://github.com/solidusio/solidus/tree/master/frontend